index.html

```diff
@@ -1,12 +1,54 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Portfolio</title>
-</head>
-<body>
-    
-</body>
-</html>
\ No newline at end of file
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>Portfolio</title>
+		<link rel="stylesheet" href="css/style.css" />
+		<link
+			href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
+			rel="stylesheet"
+		/>
+	</head>
+	<body>
+		<header class="header container">
+			<div class="header__logo">
+				<h2>My Logo</h2>
+			</div>
+			<nav class="header__nav">
+				<ul>
+					<li><a href="#">Portfolio</a></li>
+					<li><a href="#">About</a></li>
+					<li><a href="#">Contact</a></li>
+				</ul>
+			</nav>
+			<div class="header__menu">
+				<img src="assets/img/Union.svg" alt="menu" />
+			</div>
```

Comment:
- [ ] Great job on adding the hamburger button to your project :+1: :100: , but it would be even better if you wrap it using the `nav` element instead of using a `div`. This is because the nav element conveys the meaning that the enclosed content is a set of navigation links, which helps search engines and accessibility tools better understand your webpage's structure. Additionally, using the appropriate semantic element can also improve the readability and maintainability of your code, as it makes it clearer to other developers what the purpose of the enclosed content is.

Reply:
@IngDavidTM I think that a nav tag should contain navigation elements inside it, which could be added in the next stages of the project. For example in Project 7: Mobile menu (pair programming).
 Being only an SVG without functionality I thought that the most correct would be a div tag. Anyway, this pr has its static navigation menu that is invisible in the mobile version. Even so, I think it does not meet the approval requirements for this stage.